### PR TITLE
Query the new GraphQL API for user info.

### PR DESCRIPTION
**Pull request by loganfsmyth**
Depends on https://github.com/RecordReplay/backend/pull/2091

### CODE REVIEW

File: src/ui/hooks/users.ts [hunk 1]
```diff
@@ -31,44 +29,32 @@ export enum Nag {
...
-          }
-          recordings_aggregate {
-            aggregate {
-              count
-            }
-          }
           internal
           nags
+          recordings {
+            totalCount
```
Comment (loganfsmyth) [inline src/ui/hooks/users.ts:40]: This doesn't exist in the API right now, but hopefully I can remove our need for `authoredRecordingCount` entirely, then I'll just drop this part of the query.

File: src/ui/components/Account/Library.tsx [hunk 2]
```diff
@@ -57,12 +57,7 @@ function Library({ setWorkspaceId, setModal, currentWorkspaceId }: PropsFromRedu
   }, [workspaces, loading]);
 
   useEffect(() => {
-    if (
-      userInfo &&
-      userInfo.authoredRecordingCount === 0 &&
-      userInfo.nags &&
-      !userInfo.nags.includes(Nag.FIRST_REPLAY)
-    ) {
+    if (userInfo?.nags && !userInfo.nags.includes(Nag.FIRST_REPLAY)) {
```
Comment (loganfsmyth) [inline src/ui/components/Account/Library.tsx:60]: I ran some SQL on the database to set FIRST_REPLAY for all users who have recordings, so the only users without `FIRST_REPLAY` in their nags at this point should be users who have no recordings.

Comment (loganfsmyth) [reply src/ui/components/Account/Library.tsx:60]: For history's sake, this is what I ran:
```
UPDATE users SET nags = nags||'["first_replay"]'::jsonb WHERE EXISTS (SELECT * FROM recordings WHERE users.id = recordings.user_id) AND NOT (nags ?| array['first_replay']);
```

**Review by jaril — APPROVED** [review]
LGTM!

**Pull request merged**